src/mongo/delegates/Tables.class.php

```diff
@@ -87,7 +87,7 @@ public function __construct($storeName,\MongoCollection $collection,$defaultCont
 
     /**
      * Receive update from subject
-     * @param ImpactedSubject
+     * @param \Tripod\Mongo\ImpactedSubject
      * @return void
      */
     public function update(ImpactedSubject $subject)
```

Comment:
ok, we seem to be using `use` statements in some places and full qualifying in others.